### PR TITLE
multi chugging should be done if this is broken i cry

### PR DESCRIPTION
**Pull request by gritor111**
ok so if you over chug it'll auto chug the right amount and delete the right amount of items (honey jars or vault potions)

### CODE REVIEW

**Comment by XeDotCom** [comment]
what about presents and barrels

**Comment by gritor111** [comment]
i think thats how petus wants it

File: data/text.json [hunk 1]
```diff
@@ -2630,4 +2630,4 @@
       }
     }
   }
-}
+}
```
Comment (Elrynd) [inline data/text.json:2633]: Why would you commit that ?

**Comment by Iapetus-11** [comment]
closing because it didn't pass all checks and there's now conflicting files.

**Pull request closed without merging**